deploy/helm/elastic-agent/examples/kubernetes-only-logs/agent-kubernetes.yaml

```diff
@@ -0,0 +1,22 @@
+kubernetes:
+  enabled: true
+  state:
+    enabled: false
+  metrics:
+    enabled: false
+  apiserver:
+    enabled: false
+  proxy:
+    enabled: false
+  scheduler:
+    enabled: false
+  controller_manager:
+    enabled: false
+  containers:
+    logs:
+      enabled: true
+  hints:
+    enabled: false
+
+agent:
+  unprivileged: true
```

Comment:
I keep seeing this in the examples. Would it just be better to make `agent.unprivileged: true` as the default in the helm chart? This is net-new so it would be okay if it was the default.

Reply:
I started with `false` there as the capability to run rootless (properly) will be part of 8.16.0 version

Reply:
Makes sense.